LICENSES-AND-NOTICES/SPECS/data/licenses.json

```diff
@@ -2188,7 +2188,6 @@
                 "cmake-fedora",
                 "containerd",
                 "coredns",
-                "csi-driver-lvm",
```

Comment:
does cgmanifest.json need changing?

Reply:
(for both src and vendor tarballs?)

Reply:
Yes need to remove the csi entry in cgmanifest.